docs/reference/indices/open-close.asciidoc

```diff
@@ -32,3 +32,5 @@ This setting can also be changed via the cluster update settings api.
 
 Closed indices consume a significant amount of disk-space which can cause problems in managed environments. Closing indices can be disabled via the cluster settings
 API by setting `cluster.indices.close.enable` to `false`. The default is `true`.
+
+Closed index responses with `index_closed_exception` and `403` status on call to `_stats` API. Closed indices are not returned in cross index `_aliases` or `_stats` API responses.
```

Comment:
there are many more APIs that return IndexClosedException and 403 status code when executed against closed indices. For instance _search. But through the `ignore_unavailable` parameter, you can decide whether to ignore those closed indices or get an exception back instead. I would rephrase your new paragraph without mentioning any specific APIs, rather say "most APIs don't support execution against closed indices, hence `ignore_unavailable allows to control whether closed indices are skipped or cause an IndexClosedException to be returned". What do you think?